libs/template/src/lib/render-strategies/strategies/strategies-map.ts

```diff
@@ -9,6 +9,22 @@ import { getGlobalStrategies } from './global.strategy';
 
 export const DEFAULT_STRATEGY_NAME = 'local';
 
+/**
+ * @description
+ * This method returns the provided strategies as name:strategy pair
+ *
+ * Built-in Strategies:
+ *
+ * | Name      | Zone Agnostic | Render Method   | Coalescing    | Scheduling                 |
+ * | --------- | --------------| --------------- | ------------- | -------------------------- |
+ * | `local`   | ✔             | `detectChanges` | ✔             | `requestAnimationFrame`   |
+ * | `global`  | ✔             | `ɵmarkDirty`    | ❌             | ❌                        |
+ * | `detach`  | ✔             | `detectChanges` | ✔             | `requestAnimationFrame`   |
+ * | `noop`    | ✔             | ❌              | ❌             | ❌                        |
+ * | `native`  | ❌             | `markForCheck` | ❌             | ❌                        |
```

Comment:
update when we are clear what to state about Scheduling and Coalescing of the built-in renderMethods